colossus-examples/src/main/scala/colossus.examples/BenchmarkExample.scala

```diff
@@ -3,30 +3,25 @@ package colossus.examples
 import colossus.IOSystem
 import colossus.service.Callback.Implicits._
 import colossus.protocols.http._
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
 
-import org.json4s._
-import org.json4s.jackson.JsonMethods._
-import org.json4s.JsonDSL._
+object BenchmarkExample {
 
-object BenchmarkService {
-
-  implicit object JsonBody extends HttpBodyEncoder[JValue] {
-    val contentType = ContentType.ApplicationJson
-
-    def encode(json: JValue) = {
-      HttpBody(compact(render(json)))
-    }
+  implicit object JsonBody extends HttpBodyEncoder[HelloWorld] {
```

Comment:
I would generalize this a little since this isn't what you would do in the real world. Change this to `AnyRef`.